### PR TITLE
[BD] Fix typos and improve wording on the ER Model page

### PR DESCRIPTION
**Pull request by jjasferreira**

### CODE REVIEW

File: content/bd/0003-er-model.md [hunk 1]
```diff
@@ -135,16 +135,16 @@ Podemos aplicar ambas as restrições simultaneamente.
 Membro -> (Sócio, Não Sócio) | Gestor | (Regular, Não Regular)
 Sócio -> (Sócio Bronze, Sócio Silver, Sócio Gold)
```
Comment (jjasferreira) [inline content/bd/0003-er-model.md:136]: Is this meant to be here?

Comment (diogotcorreia) [reply content/bd/0003-er-model.md:136]: No :sweat_smile: 

Forgot to remove the draft

File: content/bd/0003-er-model.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ Representamos uma associação com atributos das seguinte forma:
 
 Nada nos impede de criar associações entre três ou mais entidades.
 Atenção que estas podem, por vezes, ser pouco explícitas. Alternativamente, podemos
-criar uma [agregação](#agregações), que, além de mais mais explícita, dá-nos maior
+criar uma [agregação](#agregações), que, para além de ser mais explícita, dá-nos maior
```
Comment (RafDevX) [inline content/bd/0003-er-model.md:94]:
```suggestion
criar uma [agregação](#agregações), que, para além de ser mais explícita, nos dá maior
```

File: content/bd/0003-er-model.md [hunk 3]
```diff
@@ -155,7 +152,7 @@ Podemos, tal como nas associações, aplicar restrições às especializações:
 
 - podemos obrigar a que exista a especialização
 - podemos obrigar a que a especialização seja disjunta, isto é, não pode ser também
```
Comment (RafDevX) [inline content/bd/0003-er-model.md:154]:
```suggestion
- podemos obrigar a que a especialização seja disjunta, isto é, que nenhum exemplar
```
Part 1/2 (GitHub doesn't allow suggestions spanning deleted lines)

File: content/bd/0003-er-model.md [hunk 4]
```diff
@@ -155,7 +152,7 @@ Podemos, tal como nas associações, aplicar restrições às especializações:
...
-  outra instância de outra especialização
+  uma instância de outra especialização
```
Comment (RafDevX) [inline content/bd/0003-er-model.md:155]:
```suggestion
  de uma entidade possa ser simultaneamente um exemplar de mais do que uma
  especialização (e.g., uma pessoa pode ser um _Professor_ ou um _Aluno_, mas
  nunca os dois ao mesmo tempo).
```
Part 2/2 (GitHub doesn't allow suggestions spanning deleted lines)

File: content/bd/0003-er-model.md [hunk 5]
```diff
@@ -224,7 +221,7 @@ Imaginemos que temos uma cadeia de supermercados. Cada supermercado tem um super
 que é escolhido entre os trabalhadores dessa loja. Adicionalmente, cada trabalhador tem
 de trabalhar no mínimo 6h semanais por local de trabalho.  
 Como podemos indicar no nosso Modelo E-A que o supervisor tem obrigatoriamente de trabalhar no supermercado?
-E garantir que cada trabalhador trabalha pelo menos 6h por semana em cada supermercado?
+E garantir que cada trabalhador trabalhe pelo menos 6h por semana em cada supermercado?
 Com uma Restrição de Integridade!
```
Comment (RafDevX) [inline content/bd/0003-er-model.md:225]:
```suggestion
Com Restrições de Integridade!
```

File: content/bd/0003-er-model.md [hunk 6]
```diff
@@ -260,13 +257,13 @@ ou mesmo **limitar a profundidade máxima**.
 
 ## Entidades Fracas
 
-Nem sempre conseguimos identificar uma entidade, isto é, a sua chave não é suficiente.
-Podemos ter por exemplo uma rua, um andar, etc. Cada um destes conceitos não pode existir
+Nem sempre conseguimos identificar uma entidade, isto é, por vezes, a sua chave não é suficiente para a definir.
+Podemos ter, por exemplo, uma rua, um andar, etc. Cada um destes conceitos não pode existir
 sem outro (e.g. freguesia e prédio, respetivamente).
```
Comment (RafDevX) [inline content/bd/0003-er-model.md:262]:
```suggestion
Nem sempre conseguimos identificar uma entidade, isto é, por vezes, a sua chave não é suficiente
para a especificar de forma inequivoca no universo. Podemos ter, por exemplo, uma rua, um andar,
etc. Cada um destes conceitos não pode existir sem outro (e.g. freguesia e prédio, respetivamente).
```

File: content/bd/0003-er-model.md [hunk 7]
```diff
@@ -292,7 +289,7 @@ Pegando num exemplo que ocorre no Fénix: agregamos a associação Disciplina/Se
 pelo que ganhamos granularidade na associação de professores à mesma.  
 Podemos, no exemplo da esquerda, permitir que uma execução de uma disciplina não tenha
```
Comment (RafDevX) [inline content/bd/0003-er-model.md:290]:
```suggestion
Podemos, como no exemplo da esquerda, permitir que uma execução de uma disciplina não tenha
```

**Review by RafDevX — CHANGES_REQUESTED** [review]

File: content/bd/0003-er-model.md [hunk 8]
```diff
@@ -35,26 +35,26 @@ Com este modelo conseguimos exprimir:
...
+Exemplos de entidades são _Pessoa_, _Produto_, _Compra_, etc.  
+Exemplos de atributos, no caso da entidade _Pessoa_, são _número do cartão de cidadão_,
+_nome_, _sexo_, _data de nascimento_, etc.
 
 ![Exemplo de Entidade (Pessoa) no Modelo E-A](./assets/0003-er-entity.svg#dark=3)
 
 As [**entidades**](color:orange) são representadas por um retângulo, enquanto que
 os [**atributos**](color:green) são representados por uma elipse.
-Os atributos que são chave primária representam-se a sublinhado.
+Os atributos que são chave primária distinguem-se dos restantes pelo sublinhado.
```
[the remark this answers is not in the document]
Comment (jjasferreira) [reply content/bd/0003-er-model.md:57]: Tried to rephrase it to avoid repetition of the verb "representar".

[the remark this answers is not in the document]
Comment (RafDevX) [reply content/bd/0003-er-model.md:57]:
```suggestion
Os atributos que são chave primária distinguem-se dos restantes pelo texto a sublinhado.
```
This seems fine then.

File: content/bd/0003-er-model.md [hunk 9]
```diff
@@ -72,16 +72,16 @@ Podemos representar esta associação com a seguinte notação:
 As associações têm normalmente um nome, único em todo o diagrama, que normalmente
 é um verbo.
 
-As associações são adirecionais, mas podemos utilizar os símbolos `>` e `<` para
+Para além disso, não têm direção, mas podemos utilizar os símbolos `>` e `<` para
 desambiguar alguns casos.
```
[the remark this answers is not in the document]
Comment (jjasferreira) [reply content/bd/0003-er-model.md:76]: I don't think the word "adirecional" exists in european portuguese. Might be wrong, though.

File: content/bd/0003-er-model.md [hunk 10]
```diff
@@ -260,13 +259,13 @@ ou mesmo **limitar a profundidade máxima**.
 
 ## Entidades Fracas
 
-Nem sempre conseguimos identificar uma entidade, isto é, a sua chave não é suficiente.
-Podemos ter por exemplo uma rua, um andar, etc. Cada um destes conceitos não pode existir
-sem outro (e.g. freguesia e prédio, respetivamente).
+Nem sempre conseguimos identificar uma entidade, isto é, por vezes, a sua chave não é suficiente
+para a especificar de forma inequivoca no universo. Podemos ter, por exemplo, uma rua, um andar,
```
Comment (jjasferreira) [inline content/bd/0003-er-model.md:263]:
```suggestion
Nem sempre conseguimos identificar uma entidade, isto é, por vezes, a sua chave não é suficiente
para a especificar de forma inequívoca no universo. Podemos ter, por exemplo, uma rua, um andar,
```

**Comment by RafDevX** [comment]
Adirectional doesn't seem to be a word in any language, English or Portuguese, but it's explicitly used within the course and its intended meaning is clear so I'm in favor of keeping it.

Up to @diogotcorreia.

File: content/bd/0003-er-model.md [hunk 11]
```diff
@@ -35,26 +35,26 @@ Com este modelo conseguimos exprimir:
...
+Exemplos de entidades são _Pessoa_, _Produto_, _Compra_, etc.  
+Exemplos de atributos, no caso da entidade _Pessoa_, são _número do cartão de cidadão_,
+_nome_, _sexo_, _data de nascimento_, etc.
 
 ![Exemplo de Entidade (Pessoa) no Modelo E-A](./assets/0003-er-entity.svg#dark=3)
 
 As [**entidades**](color:orange) são representadas por um retângulo, enquanto que
 os [**atributos**](color:green) são representados por uma elipse.
-Os atributos que são chave primária representam-se a sublinhado.
+Os atributos que são chave primária distinguem-se dos restantes por serem representados a sublinhado.
```
Comment (jjasferreira) [inline content/bd/0003-er-model.md:57]:
```suggestion
Os atributos que são chave primária distinguem-se dos restantes pelo texto a sublinhado.
```

File: content/bd/0003-er-model.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (RafDevX) [reply content/bd/0003-er-model.md:76]: Adirectional doesn't seem to be a word in any language, English or Portuguese, but it's explicitly used within the course and its intended meaning is clear so I'm in favor of keeping it.

Up to @diogotcorreia.

File: content/bd/0003-er-model.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (jjasferreira) [reply content/bd/0003-er-model.md]: Sure, it looks better. Thanks.

File: content/bd/0003-er-model.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (diogotcorreia) [reply content/bd/0003-er-model.md:76]: I think it's clear as it is. Even thought it might be used in the course and it's pretty easy to understand what is means, we don't need to use non-existent words

File: content/bd/0003-er-model.md [hunk 12]
```diff
@@ -132,19 +132,16 @@ Podemos aplicar ambas as restrições simultaneamente.
...
-Membro -> (Sócio, Não Sócio) | Gestor | (Regular, Não Regular)
-Sócio -> (Sócio Bronze, Sócio Silver, Sócio Gold)
-
-Por vezes duas entidades diferem por poucos ou até nenhum atributo. Quando isto
-acontece, podemos estar num caso em que podemos generalizar/especializar uma entidade noutras.
+Por vezes, duas entidades diferem por poucos ou até nenhum atributo. Quando isto
+acontece, podemos estar num caso em que conseguimos generalizar/especializar uma entidade noutras.
 Um exemplo fácil de perceber é o caso Professor/Aluno. Ambos têm um _nome_, _cartão de cidadão_,
 _data de nascimento_, etc. Podemos criar uma entidade [_Pessoa_](color:green) que tem
 duas especialidades, [_Professor_](color:orange) e [_Aluno_](color:yellow), e que tenha
```
Comment (diogotcorreia) [inline content/bd/0003-er-model.md:139]:
```suggestion
duas especializações, [_Professor_](color:orange) e [_Aluno_](color:yellow), e que tenha
```

**Review by diogotcorreia — APPROVED** [review]

**Review by RafDevX — APPROVED** [review]
